packages/ui/components/images/logo/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { View, Image, Text } from 'react-native'
+
+import { calculateImageSize } from 'ui/components/images/lib/image-size'
```

Comment:
Use relative imports for anything in a `./lib` folder - these are meant to be used internally and you might give the impression that anyone can use this outside of the `ui/components/images` dir.